go.mod

```diff
@@ -43,3 +43,5 @@ require (
 	gopkg.in/check.v1 v1.0.0-20201130134442-10cb98267c6c // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+// replace github.com/ossf/gemara => ../../ossf/gemara
```

Comment:
❓ should we keep this?

Reply:
eh, I like to leave it in there because it saves me recall time later. This is your jurisdiction if you want to pull it tho